src/components/loader-bar/loader-bar.stories.mdx

```diff
@@ -61,3 +63,20 @@ This is an example of a large LoaderBar component.
 ### LoaderBar
 
 <StyledSystemProps margin of={LoaderBar} />
+
+## Translation keys
+
+The following keys are available to override the translations for this component by passing in a custom locale object
+to the [i18nProvider](https://carbon.sage.com/?path=/story/documentation-i18n--page).
+
+<TranslationKeysTable
+  translationData={[
+    {
+      name: "loader.loading",
+      description:
+        "An alternative message to replace loading animation - for users who have requested minimal motion-based animation to be used via their OS accessibility settings.",
```

Comment:
I would also mention the other thing this is used for - the accessible name of the component itself (some users might be able to see the animated loader bar while also listening to a screenreader describing it)